platform/ios/src/MGLMapView.mm

```diff
@@ -1224,7 +1224,7 @@ - (void)handlePanGesture:(UIPanGestureRecognizer *)pan
         if (![self.delegate respondsToSelector:@selector(mapView:shouldChangeFromCamera:toCamera:)] ||
             [self.delegate mapView:self shouldChangeFromCamera:oldCamera toCamera:toCamera])
         {
-            _mbglMap->moveBy({ delta.x, delta.y });
+            _mbglMap->moveBy({ delta.x, delta.y }, mbgl::AnimationOptions{});
```

Comment:
You can probably omit `mbgl::AnimationOptions` and leave just `{}`, since there’s only one `moveBy()` method and no ambiguity as to the second parameter’s type.

As an alternative to making this change, you could also make `{}` the default for the second parameter to `mbgl::Map::moveBy()`, as you did in `mbgl::Transform::moveBy()`.